source/mainnet/net/resources/ccd-scan-transactions.rst

```diff
@@ -0,0 +1,61 @@
+.. _transactions-view:
+
+=============================
+CCDScan Transactions overview
+=============================
+
+The Transactions overview shows information about the transactions during the selected time range in the filter.
+
+.. image:: ../images/ccd-scan/ccd-scan-transactions.png
+
+|
+
+The graphs show the number of **cumulative transactions** since inception and the number of **Transactions** in a range as determined by the time selected in the filter.
+
+Below the graphs, the table contains the following:
+
+- **Transaction hash**: the first six digits of the transaction hash. You can click Copy |copy| to copy the entire hash. Click the transaction hash to see the :ref:`transaction details<home-screen-transaction>`.
+- **Status**: transaction status
+- **Age**: transaction age
+- **Type**: transaction type
+- **Block height**: the block height of the transaction.
+- **Sender**: the first six characters of the account that sent the transaction. You can click Copy |copy| to copy the entire hash. Click the account hash to see the :ref:`account details<accounts-view>`.
+- **Cost**: the transaction fee.
+
+.. _home-screen-transaction:
+
+Transaction details
+===================
+
+When you click a transaction hash, transaction information appears. The information is different depending upon whether the transaction was successful or rejected.
+
+.. image:: ../images/ccd-scan/ccd-scan-home-transaction-success.png
+
+|
+
+The transaction details shows the following.
+
+- **Transaction**: the first six characters of the transaction hash. Click Copy |copy| to copy the entire transaction hash. It also shows the transaction state.
+- **Block height/block hash**: the block height and the first six characters of the block hash. Click Copy |copy| to copy the entire block hash. Click the block hash to see the :ref:`account details<blocks-view>`.
+- **Age**: the transaction age with date/time stamp.
+- **Transaction type/cost**: the transaction type and the transaction fee.
+- **Sender**: the first six characters of the account that sent the transaction if available for that transaction. Click Copy |copy| to copy the entire hash. Click the account hash to see the :ref:`account details<accounts-view>`.
+- **Events/Reject reason**:  list of events included in the transaction. It contains more details about the transaction, including drill-through links and copy options where available. There are many different transaction types and descriptions. If you see the transaction type **Update micro CCD per Euro**, it is possible to see the new rate and the effective timestamp.
```

Comment:
Is "Update micro CCD per Euro" treated in a special way? There are other chain updates, are those not displayed in detail?